setup.py

```diff
@@ -65,11 +67,20 @@ def print_warning(*lines):
 ############
 
 
+def exec_in_terminal(command):
+    return check_output(command, stderr=PIPE).strip().decode("utf8")
+
+
 def mpi_info(cmd):
     config = mpi4py.get_config()
-    cmd_compile = " ".join([config["mpicc"], "-show"])
-    out_stream = os.popen(cmd_compile)
-    flags = out_stream.read().strip()
+
+    if sys.platform.startswith("win32"):
+        # On Windows, mpi4py is usually installed from a wheel compiled with MS-MPI.
+        # We assume that the user has installed mpiexec from the MS-MPI installer (strangely, there is no mpicc),
+        # and mpicc from mingw-w64.
+        config["mpicc"] = exec_in_terminal(["where", "mpicc"])
```

Comment:
what does `mpi4py` return when you run `get_config()` on windows?

Reply:
```
{'define_macros': 'MSMPI_VER=0xa01',
 'include_dirs': 'C:\\Program Files (x86)\\Microsoft SDKs\\MPI\\Include',
 'libraries': 'msmpi',
 'library_dirs': 'C:\\Program Files (x86)\\Microsoft SDKs\\MPI\\Lib\\x64'}
```